build.gradle.kts

```diff
@@ -67,7 +70,7 @@ enum class ReleaseChannel(
 }
 
 val isRelease = rootProject.hasProperty("release_channel")
-val releaseChannel = if (isRelease) ReleaseChannel.valueOf("release_channel"()) else ReleaseChannel.DEV_BUILD
+val releaseChannel = if (isRelease) ReleaseChannel.valueOf("release_channel"().uppercase()) else ReleaseChannel.DEV_BUILD
```

Comment:
Pretty sure this was added for debugging, and can be removed now.

Reply:
I still think it's helpful to have in general.

Reply:
I'd prefer it not be there, as I don't want this property to be set in `gradle.properties`. It should be set by CLI argument or by manually changing the build script properties (by changing the default release channel at the end of the line, for example).